config/README.md

```diff
@@ -323,7 +323,7 @@ kill_query_user: <kill_query_user_config> | optional
 heartbeat: <heartbeat_config> | optional
 
 # RetryNumber - user configuration for query retry when one host cannot respond.
-retrynumber: 0
+retry_number: 0
```

Comment:
no test failed? That suspicious. Can you make sure? @JingmaoYou

Reply:
There are some tests failed locally. But I do not know why we do not have a CI/CD here